lib/chai/interface/assert.js

```diff
@@ -1213,173 +1178,260 @@ module.exports = function (chai, util) {
   };
 
   /**
-   * ### .nestedPropertyVal(object, property, value, [message])
+   * ### .ownProperty(object, property, [message])
    *
-   * Asserts that `object` has a property named by `property` with value given
-   * by `value`. `property` can use dot- and bracket-notation for nested
-   * reference. Uses a strict equality check (===).
+   * Asserts that `object` has a direct property named by `property`.
```

Comment:
Agreed. I actually had something to the effect of "but not inherited" in a previous iteration, but ended up removing it because the way I had it worded made it sound like the assertion would fail if the object had an inherited property with the given name, whereas it's allowed to have an inherited property with the given name as long as it also has a direct property with the same name. I should be able to reword it to not sound ambiguous though.